docs/source/pipelines.rst

```diff
@@ -55,7 +55,7 @@ Pretrained Models
    EMFORMER_RNNT_BASE_LIBRISPEECH
 
 
-wav2vec 2.0 / HuBERT - SSL
+wav2vec 2.0 / HuBERT / WavLM - SSL
 --------------------------
```

Comment:
Need to update the undreline so that it matches the charlen of the title.

<img width="713" alt="Screen Shot 2022-11-28 at 5 22 16 PM" src="https://user-images.githubusercontent.com/855818/204393362-c26aca20-c319-4603-bc70-1fceb25c28e4.png">